dashboard/src/server-config/index.ts

```diff
@@ -2,10 +2,10 @@ import { ConfigInterface, throwError } from './config';
 
 // To prevent side-effects for other environments' error handling, the specific config
 // needs to be dynamically `require`d.
-if (!['production', 'staging', 'local', 'test', 'dev'].includes(process.env.IASQL_ENV ?? '')) {
-  throwError(`Invalid environment ${process.env.IASQL_ENV}`);
+if (!['production', 'staging', 'local', 'test', 'dev'].includes(process.env.NEXT_PUBLIC_IASQL_ENV ?? '')) {
+  throwError(`Invalid environment ${process.env.NEXT_PUBLIC_IASQL_ENV}`);
```

Comment:
nit - why not just use `IASQL_ENV`?

Reply:
Next doesn't make any env var that is not prefixed `NEXT_PUBLIC_` available to the front-end. We technically could use `IASQL_ENV` in the `server-config`, but since those are both going to be the same thing every time, it doesn't seem appropriate to me?

Reply:
got it